metrics/METRICS.md

```diff
@@ -0,0 +1,80 @@
+# Deploying on OpenShift
```

Comment:
We should make it clear that this is not production grade deployment for neither Grafana nor Prometheus.

Reply:
Agree.